README.md

```diff
@@ -124,12 +124,9 @@ AppToken is platform-specific. You need to generate the seprate token for Androi
   ```
 
 3. Update `app/build.gradle`:
-  ```groovy
-    plugins {
-      id 'newrelic'
-    }
-  
-  ```
```

Comment:
Hello @markrickert , thanks for bringing this to our attention. That method is the [older version](https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application) of applying plugins in Gradle, but it would still be useful to add to our documentation.

I would keep our current documentation in and add your example in below that line with something like:
````
In legacy plugin application:
```groovy
apply plugin: 'newrelic'
```
````

Reply:
Thanks for the guidance @kthai-nr. I've updated the PR as per your recommendation.